packages/sdk-manager/src/SdkManagerContext.ts

```diff
@@ -0,0 +1,190 @@
+import {
+  EVMAssetTransfer,
+  Environment,
+  EvmFee,
+  Fungible,
+  Transfer
+} from '@buildwithsygma/sygma-sdk-core';
+import { consume, createContext, provide } from '@lit/context';
+import { LitElement, html } from 'lit';
+import { customElement, property, state } from 'lit/decorators.js';
+import {
+  WalletManagerContext,
+  WalletManagerController
+} from '@builtwithsygma/sygmaprotocol-wallet-manager';
+import {
+  BaseProvider,
+  Web3Provider,
+  TransactionRequest
+} from '@ethersproject/providers';
+import { UnsignedTransaction } from '@ethersproject/transactions';
+
+export type SdkManagerStatus =
```

Comment:
should we move the types to a type file instead of having them on the same file?